repo/overrides/ocaml.nix

```diff
@@ -2,7 +2,7 @@
 # Findlib's opam package gets away with it by shadowing the `ocaml` binary on $PATH,
 # but that won't work in a nix build environment
 { makeWrapper, lib }: impl:
-impl.overrideAttrs (orig: {
+lib.overrideDerivation impl (orig: {
```

Comment:
This is the only change I'm unsure about - I recently changed from overrideDerivation to overrideAttrs because overrideDerivation works in fewer scenarios (e.g. it doesn't support further overrides). What was the error you got?

Reply:
I got a message the overrideAttrs didn't exist